apps/10address-book-ajax/script.js

```diff
@@ -2,4 +2,48 @@
 
 $(document).ready(function() {
   // You code here
+//Link to the data
+$.ajax('https://reqres-api.herokuapp.com/api/users', {
+  //after the call is made
+  success: function(users) {
+    //for the users seperating them 
+    users.forEach(function(user)  {
+      //creating the html and plugging in the data from the ajax call
+      var str =  ('<tr>' +
+        '<th>' + user.id + '</th>' +
+        '<th>' + user.first_name + '</th>' +
+        '<th>' + user.last_name + '</th>' +
+        '<td>  <a href="https://reqres-api.herokuapp.com/api/users" class=ham data-id="'+ user.id +'">view</a>  </td>' +
```

Comment:
Why do you need `https://reqres-api.herokuapp.com/api/users` in your `<a>` tag? Also you need double quotes around your `ham` class.